keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/ArtifactListener.kt

```diff
@@ -77,17 +78,16 @@ class ArtifactListener(
 
     if (repository.artifactVersions(artifact).isEmpty()) {
       val artifactSupplier = artifactSuppliers.supporting(artifact.type)
-      val latestArtifact = runBlocking {
```

Comment:
I dislike the renaming here, because it's confusing. below we have `latestVersion.version`. Could you please revert this name switching?

Reply:
Ok.